docs/experiments/categories/gcp/gcp-vm-instance-stop/gcp-instance.yaml

```diff
@@ -0,0 +1,26 @@
+## details of the gcp instance
```

Comment:
Can we also include the `SEQUENCE` env in this manifest?

Reply:
we are adding sequence in the common manifest. II have tried to use minimal information in the individual experiments